### ISSUE TITLE
Blockwishlist - small feedback

### ISSUE THREAD

**Issue by Hlavtox** [PrestaShop/PrestaShop#24530]
#### Is your feature request related to a problem?
Hi guys, just tested blockwishlist, works awesome, but I think it could get a little tweaking for better compatibility and mod friendliness.

#### Describe the solution you'd like
Custom product listing:
- I think the module should use catalog/miniatures/product.tpl, instead of it's own separate style.

![product](https://user-images.githubusercontent.com/6097524/118512782-67587e80-b733-11eb-9d4f-c1abd95a9a00.JPG)

---
Other improvements which already fixed with this PR https://github.com/PrestaShop/blockwishlist/pull/114
>**Modifying theme styling**
>There are some hardcoded styles that override theme styling. I think it should go with the flow of the theme as much as we can.
>
>Example:
>```
>.wishlist-modal .modal-cancel {
>    background: #dddddd;
>    margin-right: 10px;
>}
>```
>![modal](https://user-images.githubusercontent.com/6097524/118511598-5b1ff180-b732-11eb-9cb0-9f6e62ee3944.JPG)
>
>**Hardcoded width and heights**
>On some elements, there are hardcoded dimensions that could overflow.
>
>![Výstřižek](https://user-images.githubusercontent.com/6097524/118511976-b05c0300-b732-11eb-8923-7cbd703d07ab.JPG)

**Comment by matks** [comment]
Hello @Hlavtox is it similar to the issue https://github.com/PrestaShop/PrestaShop/issues/24392 fixed by https://github.com/PrestaShop/blockwishlist/pull/112 ?

**Comment by NeOMakinG** [comment]
@matks no, but most are fixed in https://github.com/PrestaShop/blockwishlist/pull/114 already, except the product listing template

**Comment by Hlavtox** [comment]
@NeOMakinG Awesome!

What are your thoughts on the product listing? I think it would be more consistent if it used the theme miniature.

**Comment by hibatallahAouadni** [comment]
Ping @NeOMakinG do you think you can fix the product listing issue in the same PR?

**Comment by NeOMakinG** [comment]
@hibatallahAouadni it's way harder, it will need another PR and need more reflection

**Comment by Hlavtox** [comment]
Closing, don't care anymore
